frontend/app/src/components/HoverIcon.svelte

```diff
@@ -27,4 +35,19 @@
             background-color: var(--icon-hv);
         }
     }
+
+    .unread {
```

Comment:
not too sure about this. Seems a bit of a stretch to make it part of the HoverIcon component rather than something that is just part of the slot injected in the pinned messages case. 

e.g.

```
            <HoverIcon title={$_("showPinned")}>
                <div class="pin">
                    <Pin
                        size={$iconSize}
                        color={pinnedSelected ? "var(--icon-selected)" : "var(--icon-txt)"} />
                </div>
            </HoverIcon>
```

```
    .pin {
        position: relative;
        display: grid;
        align-content: center;

        &::after {
            content: "";
            width: 10px;
            height: 10px;
            background-color: red;
            border-radius: 50%;
            position: absolute;
            bottom: -4px;
            right: -4px;
        }
    }
```

Also the combination of the icon-selected colour and the accent colour doesn't really work in light mode when the pinned panel is actually showing (arguably a problem with the icon-selected colour in light mode more than anything else).

Reply:
Yes probably is a bit of a stretch to make it part of HoverIcon.

Btw we don't show the unread dot when the pinned panel is open.